### ISSUE TITLE
Bug: False positive on prefer-const

### ISSUE THREAD

**Issue by magurotuna** [denoland/deno_lint#344]
```js
let x = 0;
x += 1;
```

This should be okay, but we're getting an error

```
thread 'rules::prefer_const::tests::hoge' panicked at 'Unexpected diagnostics found:
[
    LintDiagnostic {
        range: Range {
            start: Position {
                line: 1,
                col: 4,
            },
            end: Position {
                line: 1,
                col: 5,
            },
        },
        filename: "deno_lint_test.tsx",
        message: "\'x\' is never reassigned. Use \'const\' instead",
        code: "prefer-const",
        hint: None,
    },
]

source:
let x = 0; x += 1;
```

I'll fix it soon.
